src/utils/draftbots.js

```diff
@@ -192,7 +194,7 @@ export const getColor = (combination, picked) => {
 
 const getCoordPairs = (pack, pick, initialState) => [
   [pack - 1, initialState[0].length],
-  [pick - 1, initialState[0][0].length],
+  [pick - 1, initialState[0][pack - 1].length],
```

Comment:
Do you need `Math.max(pack - 1, 0)` here?

Reply:
Probably to be fully correct, the input should never be less than 1 though

Reply:
lets do this just in case this accidentally gets called after the last pick